system/web/context/RequestContext.cfc

```diff
@@ -740,6 +740,15 @@ component serializable=false accessors="true"{
    		return this;
 	}
 
+    /**
+    * Get the rendered content from a ColdBox integration test
+    *
+    * @return cbox_rendered_content or an empty string
+    */
+    function getRenderedContent(){
```

Comment:
I think this needs to be in the `BaseTestCase`. That's where my instinct tells me it should go.  As it does not make sense for normal MVC operation.